cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -1211,6 +1227,18 @@ const userDefinedParamsSample = `{
 			"fromDefinitionFoo": "foo",
 			"fromDefinitionBar": "bar"
 		}
+	  },
```

Comment:
I just edited the test template here by hand, @vhvb1989, but if there's some corresponding bicep file that we had that you also want me to go update, just let me know.